.github/workflows/deploy-to-aks-feast.yaml

```diff
@@ -0,0 +1,143 @@
+  schedule:
+    - cron: "17 02 * * 1"
+
+jobs:
+  deploy-solution-to-aks:
+    name: Deploy CKF + Feast solution to AKS
+    runs-on: ubuntu-24.04
+    env:
+      AZURE_CORE_OUTPUT: none
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Run YAML to Github Output Action
+        id: yaml-output
+        uses: christian-ci/action-yaml-github-output@v2
+        with:
+          file_path: ".github/dependencies.yaml"
+
+      - name: Update ENV variables from inputs if available
+        run: |
+          K8S_VERSION=${{ inputs.k8s_version || env.K8S_VERSION }}
+          echo "K8S_VERSION=${K8S_VERSION}" >> $GITHUB_ENV
+          UATS_BRANCH=${{ inputs.uats_branch || env.UATS_BRANCH }}
+          echo "UATS_BRANCH=${UATS_BRANCH}" >> $GITHUB_ENV
+
+      - name: Install CLI tools
+        run: |
+          pip install tox
+          sudo snap install juju --channel=${{ env.JUJU_VERSION }}/stable
+          sudo snap install charmcraft --classic
+          sudo snap install terraform  --channel latest/stable --classic
+          juju version
+          terraform --version
+
+      - uses: azure/login@v1
+        with:
+          creds: ${{ secrets.CHARMED_KUBEFLOW_SOLUTIONS_AKS_SERVICE_PRINCIPAL }}
+
+      - name: Create resource group and cluster
+        run: |
+          # We need to remove the dot from version
+          # due to cluster naming restrictions
+          version=${{ env.CKF_VERSION }}
+          KF_VERSION="kf-${version//.}"
+          RESOURCE_GROUP=terraform-${KF_VERSION}-ResourceGroup
+          NAME=terraform-${KF_VERSION}-AKSCluster
+          LOCATION=westeurope
+          echo "RESOURCE_GROUP=${RESOURCE_GROUP}" >> $GITHUB_ENV
+          echo "NAME=${NAME}" >> $GITHUB_ENV
+          echo "LOCATION=${LOCATION}" >> $GITHUB_ENV
+          az group create --name ${RESOURCE_GROUP} --location ${LOCATION}
+          az aks create \
+            --resource-group ${RESOURCE_GROUP} \
+            --name ${NAME} \
+            --kubernetes-version ${{ env.K8S_VERSION }} \
+            --node-count 2 \
+            --node-vm-size Standard_D8s_v3 \
+            --node-osdisk-size 100 \
+            --node-osdisk-type Managed \
+            --os-sku Ubuntu \
+            --no-ssh-key
+
+      - name: Add AKS cloud to juju and bootstrap controller
+        run: |
+          az aks get-credentials --resource-group ${{ env.RESOURCE_GROUP }} --name ${{ env.NAME }} --admin
+          juju add-k8s aks --client
+          juju bootstrap aks aks-controller
+
+      - name: Deploy and assert kubeflow-feast solution
+        run: |
+          tox -c ./modules/kubeflow-feast -vve test_deployment -- -vv -s
+
+      - name: Run UATs
+        run: |
+          git clone https://github.com/canonical/charmed-kubeflow-uats.git ~/charmed-kubeflow-uats
+          cd ~/charmed-kubeflow-uats
+          git checkout ${{ env.UATS_BRANCH }}
+          # TODO: remove --bundle when feast is promoted to stable 
```

Comment:
Is this linked in some issue ? if not maybe we should create one so we dont forget.

Reply:
good point, filed https://github.com/canonical/charmed-kubeflow-solutions/issues/39